apps/next-docs/content/components/Hero/react.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Hero
+description: Use the hero component to render a full-width banner at the top of a page.
+keywords: ['heading', 'intro', 'cta', 'banner']
+show-tabs: true
+tab-label: React
+ready: true
+figma: https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=431%3A8481
+source: https://github.com/primer/brand/blob/main/packages/react/src/Hero/Hero.tsx
+storybook: '/brand/storybook/?path=/story/components-hero--default'
+---
+
+import {HeroDescriptionVariantProp, HeroDescriptionVariantPropDefault, HeroImagePositionProp} from './react'
+
+```js
+import {Hero} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<Hero>
+  <Hero.Label>Label</Hero.Label>
+  <Hero.Heading>This is my super sweet hero heading</Hero.Heading>
+  <Hero.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus sed turpis
+    felis nam pulvinar risus elementum.
+  </Hero.Description>
+  <Hero.PrimaryAction href="#">Primary action</Hero.PrimaryAction>
+  <Hero.SecondaryAction href="#">Secondary action</Hero.SecondaryAction>
+</Hero>
+```
+
+### Images
+
+`Hero` can display an optional foreground image. Use `position` to alternate between various layouts.
+
+```jsx live
```

Comment:
FYI Another one rendering horizontally instead of vertically. Might need stack again. Wondering now if we should vertically stack by default 🤔 

<img width="1155" alt="Screenshot 2025-04-23 at 12 07 57" src="https://github.com/user-attachments/assets/251d6cf4-fd75-4fab-a36b-b06b2bf55ece" />